chains/ethereum/writer_methods.go

```diff
@@ -204,7 +204,7 @@ func (w *writer) watchThenExecute(m msg.Message, data []byte, dataHash [32]byte,
 					w.executeProposal(m, data, dataHash)
 					return
 				} else {
-					w.log.Trace("Ignoring finalization event", "src", sourceId, "nonce", depositNonce)
+					w.log.Trace("Did not find the matching finalization event, retrying...", "src", sourceId, "nonce", depositNonce)
```

Comment:
```suggestion
					w.log.Trace("Ignoring event, not a finalization event", "src", sourceId, "nonce", depositNonce)
```

Reply:
As I said in the issue, they aren't finalization events and they should still be ignored